frontend/src/pages/Organizations/Select/styles.css

```diff
@@ -2,4 +2,13 @@
     &__buttonWrapper {
         padding: 10px;
     }
-}
\ No newline at end of file
+
+    &__link {
+        color: var(--primary-text);
```

Comment:
Eventually I will create global styles for links and remove this but good for now